qa-tests-backend/src/test/groovy/SACTest.groovy

```diff
@@ -73,6 +76,22 @@ class SACTest extends BaseSpecification {
                 WAIT_FOR_VIOLATION_TIMEOUT)
         assert waitForViolation(DEPLOYMENT_QA2.name, "Secure Shell (ssh) Port Exposed",
                 WAIT_FOR_VIOLATION_TIMEOUT)
+
+        // Make sure each deployment has a risk score.
+        def deployments = DeploymentService.listDeployments()
+        deployments.each {
+            def depID = it.id
+            try {
+                withRetry(30, 2) {
+                    assert DeploymentService.getDeploymentWithRisk(depID).hasRisk()
+                }
+            } catch (Exception e) {
+                if (strictIntegrationTesting) {
```

Comment:
What `strictIntegrationTesting` is for? If it's on we just rethrow exception but when it's fase we pass meanignful exception?

Reply:
Hm I took this from `ImageScannigTests`, but I can remove it also if it doesn't really help at all.